routes/index.js

```diff
@@ -4,16 +4,20 @@ const app = express()
 app.use('/auctions', require('./auctions.js'))
 app.use('/auth', require('./auth.js'))
 app.use('/badges', require('./badges.js'))
+app.use('/badges', require('./badges.js'))
 app.use('/challenges', require('./challenges.js'))
 app.use('/contributions', require('./contributions.js'))
+app.use('/contributions', require('./contributions'))
```

Comment:
This change is because you've removed the .js at the end, no point of keeping duplicates

Reply:
Done